src/main/java/com/iota/iri/service/snapshot/impl/SnapshotServiceImpl.java

```diff
@@ -113,30 +118,43 @@ public void replayMilestones(Snapshot snapshot, int targetMilestoneIndex) throws
                 if (currentMilestone != null) {
                     StateDiffViewModel stateDiffViewModel = StateDiffViewModel.load(tangle, currentMilestone.getHash());
                     if(!stateDiffViewModel.isEmpty()) {
```

Comment:
FYI,
When we refactor the DB this will have to also have a null check.
As to right now it doesn't matter

Reply:
yeah once we refactor we have to look for usage anyway.